presto-spi/src/main/java/com/facebook/presto/spi/plan/CteConsumerNode.java

```diff
@@ -73,13 +77,20 @@ public PlanNode replaceChildren(List<PlanNode> newChildren)
     {
         // this function expects a new instance
         checkArgument(newChildren.size() == 0, "expected newChildren to contain 0 node");
-        return new CteConsumerNode(getSourceLocation(), getId(), getStatsEquivalentPlanNode(), originalOutputVariables, cteName);
+        return new CteConsumerNode(getSourceLocation(), getId(), getStatsEquivalentPlanNode(), originalOutputVariables, cteName, originalSource);
     }
 
     @Override
     public PlanNode assignStatsEquivalentPlanNode(Optional<PlanNode> statsEquivalentPlanNode)
     {
-        return new CteConsumerNode(getSourceLocation(), getId(), statsEquivalentPlanNode, originalOutputVariables, cteName);
+        // Source is always the statsEquivalent planNode
```

Comment:
What does this mean?

Reply:
Ohh I will remove this comment, its not intended thanks